_oasis

```diff
@@ -22,6 +25,7 @@ Library "cephes"
   BuildTools:       cephes_bindgen
 
 Executable "cephes_bindgen"
+  Build$:           flag(tests)
```

Comment:
On Sun, 26 Jul 2015 15:21:09 -0700, Leonid Rozenberg wrote:
> 
> In _oasis:
> 
> > @@ -22,6 +25,7 @@ Library "cephes"
> >    BuildTools:       cephes_bindgen
> >
> >  Executable "cephes_bindgen"
> > +  Build$:           flag(tests)
> 
> I can understand why the above are relevant, but how does this influence the
> opam package generation? Sorry, I've just never used oasis2opam before and
> don't understand the details.

1. It avoids a warning of oasis2opam.

2. If the executable is built — even if not installed — its dependencies are mandatory (not really relevant in this case as it requires no additional deps).

3. Why build something that you will not install (OK for a developer but for a end user?), isn't it a waste of computing power? ;-)